packages/experimental/Expander/windows/ReactNativeExpander/ExpanderViewManager.cpp

```diff
@@ -85,49 +84,65 @@ namespace winrt::ReactNativeExpander::implementation {
 
     ConstantProviderDelegate ExpanderViewManager::ExportedCustomDirectEventTypeConstants() noexcept {
         return [](winrt::IJSValueWriter const& constantWriter) {
-            WriteCustomDirectEventTypeConstant(constantWriter, "onChange");
             WriteCustomDirectEventTypeConstant(constantWriter, L"Collapsing");
             WriteCustomDirectEventTypeConstant(constantWriter, L"Expanding");
         };
     }
 
     // IViewManagerWithChildren
-    int32_t ExpanderViewManager::ReplaceChild(winrt::Windows::UI::Xaml::FrameworkElement const& parent, winrt::Windows::UI::Xaml::UIElement const& oldChild, winrt::Windows::UI::Xaml::UIElement const& newChild) noexcept {
-        return 0;
+    void ExpanderViewManager::ReplaceChild(winrt::Windows::UI::Xaml::FrameworkElement const& parent, winrt::Windows::UI::Xaml::UIElement const& oldChild, winrt::Windows::UI::Xaml::UIElement const& newChild) noexcept {
+        // This implementation is not complete; still need to test/tweak
+        // auto expander = parent.as<xaml::Controls::ContentPresenter>().Content().as<Microsoft::UI::Xaml::Controls::Expander>();
+
+        // if (oldChild == expander.Header()) {
+        //     expander.ClearValue(Microsoft::UI::Xaml::Controls::Expander::HeaderProperty());
+        //     expander.Header(newChild);
+        // }
+        // else if (oldChild == expander.Content()) {
+        //     expander.ClearValue(xaml::Controls::ContentControl::ContentProperty());
+        //     expander.Content(newChild);
+        // }
     }
 
     void ExpanderViewManager::AddView(winrt::Windows::UI::Xaml::FrameworkElement const& parent, winrt::Windows::UI::Xaml::UIElement const& child, int64_t index) noexcept {
-        auto contentPresenter = parent.try_as<winrt::Windows::UI::Xaml::Controls::ContentPresenter>();
-        auto expanderWrapper = contentPresenter.Content();
-        auto expander = expanderWrapper.try_as<Microsoft::UI::Xaml::Controls::Expander>();
-
-        if (auto content = child.try_as<winrt::Windows::UI::Xaml::FrameworkElement>()) {
-            if (index == 0) {
-                expander.Header(content);
-            }
-            else if (index == 1) {
-                expander.Content(content);
+        auto expander = parent.as<xaml::Controls::ContentPresenter>().Content().as<Microsoft::UI::Xaml::Controls::Expander>();
+        if (index == 0) {
+            if (auto newContent = expander.GetValue(Microsoft::UI::Xaml::Controls::Expander::HeaderProperty())) {
```

Comment:
got confused by the naming here: the logic is correct, but I'd change `newContent` to `header/currentHeader`